pyprep/utils.py

```diff
@@ -46,6 +46,8 @@ def _mat_round(x):
 def _mat_quantile(arr, q, axis=None):
     """Calculate the numeric value at quantile (`q`) for a given distribution.
 
+    Currently only supports 1-D or 2-D arrays.
```

Comment:
I would put this note under the `arr` parameter description. And also add a short check right at the beginning of the fun:

```python
if arr.ndim not in [1, 2]:
    raise ValueError(...)
```

Reply:
Fixed in the latest commit!